shared/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig.json",
-  "include": ["src"],
+  "include": ["src", "build"],
```

Comment:
Are you sure that it is needed?

Reply:
Initially, I added this change to modify UserSignUpResponseDto to add the token,  but I would get an error: 
`Parsing error: Cannot read file '/users/vlad_d/desktop/bsa-2023-calmpal/shared/shared/tsconfig.json'.`
The only solution I found is to include the build directory that contained the UserSignUpResponseDto type. 
If there is abetter way to fix it I'll gladly change it.

Reply:
That is the issue with eslint plugin and it won't influenced to linting the project. Typescript should check only src/ folder)

Reply:
So I can safely remove, then? Okay, thank you!